src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -6603,9 +6603,11 @@ export interface NS {
    * print(ServerMaxMoney);
    * print(HackExpGain);
    * ```
-   * @returns Object containing the current BitNode multipliers.
+   * @param BitNode - BitNode you want the multipliers for.
+   * @param lvl - The level of the BitNode (for BN12)
+   * @returns Object containing the current (or supplied) BitNode multipliers.
    */
-  getBitNodeMultipliers(): BitNodeMultipliers;
+  getBitNodeMultipliers(BitNode, lvl): BitNodeMultipliers;
```

Comment:
This documentation should show the types for these params and should show that they are optional. Variable names should not generally be capitalized.

```ts
  /* ...
   * @param bitNode - The BitNode to get multipliers for. Defaults to the current BitNode.
   * @param lvl - The level of the BitNode (for BN12). Defaults to 1.
   * @returns Object containing the current (or supplied) BitNode multipliers. */
getBitNodeMultipliers(bitNode?: number, lvl?: number): BitNodeMultipliers;
```